index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav class="navbar" id="nav-bar">     
```

Comment:
- [ ] Kindly use the comments politely, as suggestions instead of orders, and always make sure that `why` you want the changes like why the website should have the menu fixed. Kindly edit this portion and make changes.